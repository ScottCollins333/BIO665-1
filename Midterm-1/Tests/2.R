source("2.R")

# This generates some random numerical data.
# Setting a seed ensures the same numbers are generated each time.
set.seed(1)
data1 = matrix(runif(200), nrow=10)
rownames(data1) = paste("Drug", 1:10, sep="_")
colnames(data1) = paste("CellLine", 1:20, sep="_")

# This generates another matrix of random data
data2 = matrix(runif(500), nrow=5)
rownames(data2) = paste("Drug", 1:5, sep="_")
colnames(data2) = paste("CellLine", 1:100, sep="_")

print(evaluateDrugs(data1))
print(evaluateDrugs(data2))