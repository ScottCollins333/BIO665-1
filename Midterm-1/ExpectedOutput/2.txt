$Drug_1
[1] "CellLine_2"

$Drug_2
[1] "CellLine_10"

$Drug_3
[1] "CellLine_14"

$Drug_4
[1] "CellLine_3"

$Drug_5
[1] "CellLine_6"

$Drug_6
[1] "CellLine_12"

$Drug_7
[1] "CellLine_3"

$Drug_8
[1] "CellLine_16"

$Drug_9
[1] "CellLine_7"

$Drug_10
[1] "CellLine_1"

$Drug_1
[1] "CellLine_59"

$Drug_2
[1] "CellLine_67"

$Drug_3
[1] "CellLine_58"

$Drug_4
[1] "CellLine_42"

$Drug_5
[1] "CellLine_17"

